src/main/java/edu/matc/covidPulse/controller/AllService.java

```diff
@@ -1,19 +1,58 @@
 package edu.matc.covidPulse.controller;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import edu.matc.covidPulse.entity.CountyCovidData;
+import edu.matc.covidPulse.entity.CountyFips;
+import edu.matc.covidPulse.persistence.GenericDao;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.transaction.Transactional;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.Response;
+import java.util.*;
 
-@Path("/allData")
+@Path("/all")
+@Transactional
 public class AllService {
+
+    //private GenericDao countyFipsDao;
+    private GenericDao countyCovidDataDao;
+    ObjectMapper mapper;
+
+    private final Logger logger = LogManager.getLogger(this.getClass());
+
+    public AllService() {
+        countyCovidDataDao = new GenericDao<>(CountyCovidData.class);
+        mapper = new ObjectMapper();
+    }
+
     @GET
-    @Path("/allData")
-    @Produces("text/plain")
+    @Produces("application/json")
     public Response getAllData() {
-        String data = "This will be all of the data";
-        return Response.status(200).entity(data).build();
+
+        //countyFipsDao = new GenericDao(CountyFips.class);
+
+        List<CountyCovidData> countyCovidDataList = countyCovidDataDao.getAll();
+
+        String json = "";
+
+        try {
+
+            mapper.enable(SerializationFeature.INDENT_OUTPUT);
```

Comment:
I don't think we need this setting on the mapper. I think it's preferred for us to return raw JSON without formatting...many tools (including your browser) should be able to pretty-print these on their own by reading the ContentType header (which the @ Produces annotation is setting as "application/json")